iac-extensions/kubernetes/src/main/java/org/sonar/iac/helm/utils/HelmFilesystemUtils.java

```diff
@@ -111,4 +108,32 @@ private static String resolveToInputFile(Path helmDirectoryPath, InputFile addit
   public static String normalizeToUnixPathSeparator(String filename) {
     return filename.replace('\\', '/');
   }
+
+  public static String normalizeToRuntimePathSeparator(String filename) {
+    return filename.replace('\\', File.separatorChar)
+      .replace('/', File.separatorChar);
+  }
+
+  public static boolean includeFile(String name) {
+    var normalized = normalizeToRuntimePathSeparator(name);
+    if (INCLUDED_EXTENSIONS.stream().anyMatch(normalized::endsWith)) {
+      var parent = new File(".");
```

Comment:
Instead of using `.` (current directory), the location of the tar file should be used.

Reply:
Or alternatively checking for following patterns in filename: `../`, `/..`, `..\`, `\..`, `~`, `./`, `.\`